src/microbe_stage/MicrobeAI.cs

```diff
@@ -335,6 +336,8 @@ private void FleeFromPredators(Random random, Vector3 predatorLocation)
 
         // No matter what, I want to make sure I'm moving
         SetMoveSpeed(Constants.AI_BASE_MOVEMENT);
+
+        SprintIfActive();
```

Comment:
Should the AI stop sprinting in certain cases?

Reply:
In the first iteration, I would suggest not. Sprint never lasts that long anyway, and an AI slowing down too soon and getting eaten will be more noticeable than an AI wasting a little ATP running away from something for a little too long.

Reply:
I guess I can open an issue about making the AI decide to start / stop sprinting in a more smart way...